haxepunk/graphics/BitmapText.hx

```diff
@@ -254,14 +227,9 @@ class BitmapText extends Graphic
 	function parseText():Void
 	{
 		// clear current opcode list
-		while (opCodes.length > 0)
-		{
-			opCodes.pop();
-		}
-		while (_scaleStack.length > 0)
-		{
-			_scaleStack.pop();
-		}
+		HXP.clear(opCodes);
```

Comment:
`HXP.clear` uses array.splice. Since splice returns an array, this results in allocating an array on every call, even if the caller isn't using the return value. There's a Haxe issue for this: https://github.com/HaxeFoundation/haxe/issues/4238. Here the array shouldn't be too large so popping the contents out will reduce GC activity.